src/view/gui/CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ SET( GUI
   fSBRTAnalysisDialog
   fTexturePipelineDialog
   AppearancePage
+  DragAndDropSharingDockWidget
```

Comment:
Another name would be better such as "QCustomDockWidget" or "CaptkDockwidget". As of now we are just handling drag and drop, but if in the future we extend it further, the current name will be a misnomer.

Reply:
I just pushed a commit that changes this to CaptkDockWidget.